src/API/Status/PreviewCard.vala

```diff
@@ -121,6 +129,47 @@ public class Tuba.API.PreviewCard : Entity, Widgetizable {
 		}
 	}
 
+	#if CLAPPER_0_10
+		private bool enhancer_check (string scheme, string? t_host = null) {
+			if (!Clapper.WITH_ENHANCERS_LOADER) return false;
```

Comment:
Just for information, this is fine cause it might save you from calling `.substring (4)` below, so you can leave it as-is. But other than that it is perfectly safe for applications to call `Clapper.get_global_enhancer_proxies ()` - in this case returned `GListModel` will simply have zero elements and `for` loop would not execute.